dictionaries/humanist_errors.en.yml

```diff
@@ -14,3 +14,5 @@
 :zero_division_error:
   :divide_by_zero: "Science has not figured out how to divide by zero yet. Damn you science!!!"
 
+:load_error:
+	:no_file: "You tried to load a file that doesn't exist - at least, not where you think it is!"
```

Comment:
Looks like we just need a little of white space clean up here.  Git is complaining about the lack of a new line at the end of the file,  and there is tab character at the beginning of the line instead of 2 spaces.